package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -148,7 +148,7 @@
    =========== ==============  =========== ====================================
    group       donor           acceptor    comments
    =========== ==============  =========== ====================================
```

Comment:
Please extend the three `===` lines so that they cover your text. Alternatively, you can also break the line.

Although [simple tables](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#simple-tables) may overflow on the right border, it is recommended that the borders encompass all text:
> The rightmost column is unbounded; text may continue past the edge of the table (as indicated by the table borders). However, it is recommended that borders be made long enough to contain the entire text.